lib/tasks/update_aliquots.rake

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+namespace :update_aliquots do
```

Comment:
Personally I would call this `aliquots` and then call the task `add_missing_uuids`.  That makes it very clear when this gets called like `bundle exec rake aliquots:add_missing_uuids`

This is consistent with the way rails uses tasks like `db:setup` and `assets:clobber`.  You have a noun on the left and a verb on the right.

Reply:
Agree, it makes it more clear 👍